### DISCUSSION TITLE
[Skinning] Lazer skin features on legacy skins

### DISCUSSION THREAD

**Discussion by Hexality** [ppy/osu#14938] [Q&A]
Hi, I have 2 questions:
The first question is about the slider tracks
> Will there be an option in the SliderStyle (skin.ini) to allow lazer skin-like slider tracks on legacy skins? for an example, the ability to remove the gradient from the slider?

The second question is 
> Will we be able to separate the scorebar-bg from the scorebar-colour thing in a way where we can for an example, add 2 scorebar-color and only 1 scorebar-bg on the ui?
It would be really cool if I could remove the scorebar bg from my mirrored lifebar, since the elements on the bg are overlapping and creating such a weird appearence xD.
![image](https://user-images.githubusercontent.com/17398632/135737347-51feb30a-b3f3-439a-8f27-23260cadfe47.png)

**Comment by bdach** [comment]
The answer to both questions is likely yes but it is too early to give specifics. There will be a new skinning system to allow this that will no longer be based on `skin.ini` and the particulars of how that will work are not yet fully fleshed out.
